modules/swagger-codegen/src/main/resources/python/api_doc.mustache

```diff
@@ -25,10 +25,14 @@ import {{{packageName}}}
 from {{{packageName}}}.rest import ApiException
 from pprint import pprint
 {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
```

Comment:
Where does the `#isBasic` flag come from? Should we be updating it in here somewhere or it comes from our api spec?

Reply:
API spec, I believe. I'm going to check the generated docs to make sure it works though.

Reply:
Yeah, it works 👌